js/wp-seo-shortcode-plugin.js

```diff
@@ -0,0 +1,281 @@
+/* global console */
+/* global tinyMCE */
+/* global wpseoShortcodePluginL10n */
+/* global ajaxurl */
+/* global YoastSEO */
+/* global yoastShortcodePlugin */
+/* jshint -W097 */
+/* jshint -W003 */
+'use strict';
+
+/**
+ * The Yoast Shortcode plugin parses the shortcodes in a given piece of text. It analyzes multiple input fields for shortcodes which it will preload using AJAX.
```

Comment:
If this is a public object (and it is) we should document the object properties as well.